### PR TITLE
[COURSE] Add UMich EECS 498-007: Deep Learning for Computer Vision

### PR DESCRIPTION
**Pull request by xzhseh**
This PR introduces the addition of the computer vision course `EECS 498-007` offered by the University of Michigan (UMich), taught by Justin Johnson.

To mention, Justin Johnson is also the main contributor and maintainer of course `CS231N` when he's still pursuing his Ph.D. in Stanford.

Designed with beginners in mind, `EECS 498-007` serves as an excellent resource and hands-on practice for those new to computer vision. The course does not require extensive prerequisites, making it accessible for a wide range of students.

I highly recommend this course to any student harboring an interest in computer vision. Your journey into this fascinating field could very well start here.

### CODE REVIEW

**Comment by xzhseh** [comment]
Also, if the Chinese version looks good, I'll be glad to translate this to English :)

File: docs/深度学习/EECS498-007.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# EECS 498-007 / 598-005: Deep Learning for Computer Vision
```
Comment (PKUFlyingPig) [inline docs/深度学习/EECS498-007.md:1]: UMich EECS

**Comment by PKUFlyingPig** [comment]
The Chinese version LGTM, look forward to your English version.

**Comment by xzhseh** [comment]
I've added the English version for `EECS498-007` and fixed several typos for the original version, also improved the overall format :)

**Pull request merged**